### PR TITLE
[V6r17] better bucket size

### PR DESCRIPTION
**Pull request by zmathe**
I changed the buckets in order to support queries which require more than one year data. The maximum buckets size is 7 weeks.

### CODE REVIEW

**Comment by fstagni** [comment]
OK, just correct the failing test.

**Comment by zmathe** [comment]
Done

File: MonitoringSystem/DB/test/Test_monitoringdb.py [hunk 1]
```diff
@@ -74,12 +74,13 @@ def test_determineBucketSize( self ):
...
     result = client._determineBucketSize( 1427038370, 1458660770 )
     self.assert_( result['OK'] )
-    self.assertEqual( result['Value'], ( '1w', 86400 * 7 ) )
+    self.assertEqual( result['Value'], ( '7w', 86400 * 7 * 7 ) )
 
 
     result = client._determineBucketSize( 1300807841, 1458660641 )
-    self.assert_( result['Message'] )
-
+    self.assert_( result['OK'] )
```
Comment (fstagni) [inline MonitoringSystem/DB/test/Test_monitoringdb.py]: Any idea why on this assertion changed?

Also, `assert_` is now deprecated in favor of `assertTrue()`: https://docs.python.org/2/library/unittest.html#deprecated-aliases

Comment (zmathe) [inline MonitoringSystem/DB/test/Test_monitoringdb.py]: yes, it was able to determine the bucket size only one year and now I extended to two years.

**Pull request merged**